index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+   <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <!-- Navbar -->
+
+    <header>
+       <div class="nav-container">
+        <div class="bars">
+            <div class="time">
+                <img src="./images/↳-Time.png" alt="Picture of the current time">
+            </div>
+               <div class="bar">
+                   <img src="./images/Group-3.png" alt="">
+               </div>
+        </div>
+          <nav class="nav-bar">
+        <div class="logo">
+            <h1>Welcome</h1>
+        </div>
+        <div class="harmburger">
+         <img src="./images/Union.svg" alt="Toggle button to display hidden nav items">
+        </div>
+          </nav>
+       
+       </div>
+         <div class="header-container">
+             
+            <div class="header-content">
+                <p class="intro">Hey there. I'm Oyinlade <br>
+                    <span class="hello-span">I'm a software developer</span></p>
+               <p class="bio"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+               </div>
+         </div>
+           <div class="social-media-icon">
+             
+               <img src="./images/twitter.png" alt="">
+               <img src="./images/Linkedin-icon.svg" alt="">
+               <img src="./images/Messenger.png" alt="">
+               <img src="./images/github.png" alt="">
+               <img src="./images/two-fingers.svg" alt="">
+            
+           </div>
```

Comment:
**[REQUIRED CHANGES]**

- To enforce the use of HTML5 semantic tag, kindly use a  `<ul>` tag here as the `<ul>` tag defines an unordered list used together with the  `<li>` tag. you could do this to achieve that:
```

<ul>
  <li>
    <a href="">
      <img src="" alt="">
    </a>
  </li>
</ul>
```